docs/connected_orgs.rst

```diff
@@ -1,14 +1,76 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you first need to have a Connected App configured in a persistent Salesforce org.
```

Comment:
This isn't accurate (in most cases you can just use our default connected app). I'd give this section an intro like this:

In addition to creating scratch orgs [link to scratch org section], you can connect persistent orgs to your CumulusCI project so that you can run tasks and flows on them. This supports use cases such as deploying to a Developer Edition org in order to release a package version, or installing to a sandbox for user acceptance testing.